spec/vorpal/unit/config_builder_spec.rb

```diff
@@ -43,7 +43,7 @@ class Tester; end
     end
 
     it 'does not redefine constants' do
-      stub_const('A::B::C::TestDB', 1)
```

Comment:
I was expecting this question. When you use `stub_const` RSpec will remove the stub after the test runs by undefining it; however, now we are explicitly calling `A::B::C.undefine_test_db_class` in an after RSpec block which undefines the constant. Using `stub_const` will make RSpec try to remove the constant after we already undefine it, provoking an error.